org.panorama-research.mobstr.tim/src/org/panorama/research/mobstr/tim/MobstrTIMAdapter.java

```diff
@@ -78,6 +78,10 @@ public EObject createModel() {
 			traceTypes.add(TimPackage.eINSTANCE.getRelatedTo());
 			traceTypes.add(TimPackage.eINSTANCE.getSpecifiedByRequirement());
 			traceTypes.add(TimPackage.eINSTANCE.getUsesAsEvidence());
+			
+			// OFFIS Extensions
```

Comment:
Same as above. Is this a meaningful comment if someone who is not familiar with the history of the project looks at that code in a year or two?

Reply:
I have removed that comment